apache/superset
# chore(Dashboard): Highlight errored filters on the left pane of the Native Filters form plus several enhancements

### SUMMARY
This PR introduces several enhancements to the native filters form and some new functionalities:

- It shows [untitled] on the left pane when no name is entered. This is consistent with other parts of the platform
- It shows an error indication on the left pane when an error is generated on the fly or on save. This drags the attention of the user to the right filters where an action is required
- It restructures a little bit the form using the Antdesign `Form.Item` wherever possible. However, the constraints of this implementation are huge and I advocate for a full refactor _asap_

### AFTER

https://user-images.githubusercontent.com/60598000/136043592-7eaa2ca3-475a-40e0-87a7-35a40691d1c2.mp4

### TESTING INSTRUCTIONS
1. Open a Dashboard with native filters enabled
2. Fill in the form and observe the behavior when an error is generated

### ADDITIONAL INFORMATION
